README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="/logo/logotype-horizontal.png"></p>
```

Comment:
Could you add an `alt` attribute with an appropriate string, e.g. `Logo courtesy of @Tobaloidee` or with your preferred name. It is only right that we give you credit for this.

Reply:
You can also use the Markdown syntax here, i.e. `![Logo by Tobaloidee](logo.png)`